_posts/2021-04-27-Optimising-ux-for-software.markdown

```diff
@@ -7,70 +7,65 @@ author: Lucy Sharp
 
-## DevOps
+## Wait, we need to know what DevOps is too...
 
-For some, DevOps is a buzzword. They trott it out and throw it around in conversation. For others, DevOps is the How. It’s a practice. A way of working. Or the “foreground to a great cult,” to quote Hamish.
+For some, DevOps is a buzzword. They trott it out and throw it around in conversation. For others, DevOps is the How. It’s a practice. A way of working. Or the “foreground to a great cult,” to quote [Hamish](https://www.linkedin.com/in/hamishtedeschi/).
 
-So what is DevOps? When I spoke to Nick about this, he could not help himself but take me back to Toyota and manufacturing. Despite my protests of, “I know this,” Nick persisted and I’m glad he did. What I learnt from our conversation was twofold.
+So what is DevOps? When I spoke to [Nick](https://www.linkedin.com/in/nickvjenkins/) about this, he could not help himself but take me back to Toyota and manufacturing. Despite my protests of, “I know this,” [Nick](https://www.linkedin.com/in/nickvjenkins/) persisted and I’m glad he did. What I learnt from our conversation was twofold.
 
 Firstly, DevOps is about removing waste to support the flow of value through a system. It’s also about automation, feedback loops and breaking down silos. DevOps originated from a place of friction, where development and operations had competing challenges impeding delivery. By understanding these dependencies and constraints, silos were removed to create a shared understanding and teams were better equipped to deliver software.
 
-Secondly, I learnt the concept of value in DevOps. Features in software either add value or add waste. Waste being a redundant functionality that is not useful.
+Secondly, I learnt the concept of value in DevOps. Features in software either add value or are waste. Waste being a redundant functionality that is not useful.
 <br /><br />
 
-## Design in DevOps
-
-The flow of value through a system is impeded by constraints such as silos. Silos being teams not working together and no shared understanding of the problem space. When UX is siloed from development teams, designers lack shared understanding about software constraints, and therefore the system in which a user experience operates in; decisions are made with partial context; impeding this flow. As Amy Chess considers in her article on [UX and Domain Driven Design](https://www.uxmatters.com/mt/archives/2019/03/platform-ux-domain-driven-design-concepts-in-enterprise-ux.php) UX and Domain Driven Design “it’s important to understand these relationships from a UX perspective because they illuminate how interactions between enterprise systems can exert significant impacts on the user experience.” There is value in bringing UX together with DevOps to accelerate technology delivery.
-
-How does UX fit within the DevOps loop? The below diagram (fig 1.0) is based on how designers work with developers at Mechanical Rock.
+## Ok, we're ready. Here are my three reasons:
```

Comment:
Three reasons for what?

Headings should be a signpost for the reader, and I don't know where I'm heading.